### PR TITLE
updated version / eslint / fixed linting

### PR DESCRIPTION
**Pull request by WaleedAshraf**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,18 +16,17 @@
     "lifecycle",
     "component"
   ],
-  "author": "Wolfram Fischer <wolfram.fischer@relayr.de>",
+  "author": "Relayr Nodejs Team - Wolfram Fischer",
```
Comment (WaleedAshraf) [inline package.json]: I kept "Wolfram Fischer" because he was the actual creator? Maybe we can skip it.

Comment (Richard-Boyewa) [reply package.json]: Is there any email attributed to relayr's engineering team? An email needs to be quoted incase someone needs to contact the author (in this case Relayr).

Comment (WaleedAshraf) [reply package.json]: they can always open issue in this repo if needed.

Comment (Richard-Boyewa) [reply package.json]: Ok.

**Comment by WaleedAshraf** [comment]
@relayr/nodejs we'll need to update the version to 1.1.0 in all node repos. This can be done when we fix this issue: https://relayr.atlassian.net/browse/C20-3105

**Comment by bernardpletikosa** [comment]
No need to update anything before we have a fix for zerostep that will solve MQTT issues.

**Pull request merged**